assets/app/views/directives/duration.html

```diff
@@ -0,0 +1,2 @@
+<span ng-if="round" data-timestamp="{{timestamp}}" class="duration-rounded">{{timestamp | duration : null : true : 1}}</span>
```

Comment:
I was trying to avoid the seconds ticking in the events table when under a minute. I only refresh duration-rounded every 30 seconds.

We'd also need to read the duration attribute from the interval in app.js, but that might not be too hard.